test/verify/check-storage-basic

```diff
@@ -61,7 +61,12 @@ class TestStorageBasic(StorageCase):
         self.content_row_wait_in_col(1, 2, "ext2 filesystem")
 
         self.content_tab_expand(1, 1)
-        b.assert_pixels("#detail-content", "partition", ignore=["dt:contains(UUID) + dd"])
+        b.assert_pixels("#detail-content", "partition", ignore=[
+            # UUID is not stable
+            "dt:contains(UUID) + dd",
+            # browsers/harfbuzz don't render "ext2 filesystem" label reliably
```

Comment:
I would ignore the column that this is in, like we did in https://github.com/cockpit-project/cockpit-machines/pull/564. My medium-strong hunch is that this has nothing to do with the actual text, and everything with the size of the columns.

Reply:
Hmm, interesting -- all the screenshots that I've looked at had the initial "ext" and the final "m" just fine, just the "2" and some letters in "filesyste" are jumbled. But I'm fine with changing it.

Reply:
Done.